services/venice-controller/src/test/java/com/linkedin/venice/controller/TestDisabledPartitionEnablerService.java

```diff
@@ -30,11 +35,32 @@ public void testCleanupBackupVersion() throws Exception {
     doReturn(true).when(controllerConfig).isEnableDisabledReplicaEnabled();
     doReturn(clusters).when(config).getClusters();
     TestMockTime time = new TestMockTime();
-    DisabledPartitionEnablerService service = new DisabledPartitionEnablerService(admin, config, time);
+    DisabledPartitionEnablerService service = new DisabledPartitionEnablerServiceForTesting(admin, config, time);
 
     service.startInner();
+    Assert.assertTrue(latch.await(1, TimeUnit.SECONDS));
     verify(admin, atLeast(1)).enableDisabledPartition(clusterName, "", true);
+    service.stopInner();
```

Comment:
Not an expert on Mockito, do you think [VerificationWithTimeout](https://site.mockito.org/javadoc/current/index.html?org/mockito/verification/VerificationWithTimeout.html) help in this case?

Reply:
Yes! I think so! That probably require less code changes.. Let me incorporate that.

Reply:
Done! @lluwm